src/locale/locales/fr/messages.po

```diff
@@ -3641,11 +3630,11 @@ msgstr "Plus anciennes réponses en premier"
 
 #: src/components/StarterPack/QrCode.tsx:69
 msgid "on"
-msgstr ""
+msgstr "sur"
 
 #: src/lib/hooks/useTimeAgo.ts:81
 msgid "on {str}"
-msgstr ""
+msgstr "à {str}"
```

Comment:
Nope! A nasty one: here, `{str}` is a "long date", so it should be `le`, as in `le lundi 4 décembre`, and not `à lundi 4 décembre`.

```suggestion
msgstr "le {str}"
```

Reply:
(for reference, as it wasn't very clear in my comment: we use `le` in front of dates and `à` in front of times. So we would write `envoyé à 14:36` but `envoyé le lundi 4 décembre`. Of course, we could also write `envoyé hier` or `envoyé demain` — and not ~~`envoyé le hier`~~ or ~~`envoyé le demain`~~; and sometimes, we can ditch the `le` in front of the date, like in `envoyé lundi 4 décembre`, but it's tricky to generalize.)